### PR TITLE
Fix G-Max Depletion

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/moves.ts [hunk 1]
```diff
@@ -6303,10 +6303,11 @@ export const Moves: {[moveid: string]: MoveData} = {
...
-						const ppDeducted = pokemon.deductPP(move.id, 4);
+						const ppDeducted = pokemon.deductPP(move.id, 2);
 						if (ppDeducted) {
 							this.add("-activate", pokemon, 'move: G-Max Depletion', move.name, ppDeducted);
-							return;
+							// Don't return here because returning early doesn't trigger
+							// activation text for the second Pokemon in doubles
 						}
 					}
 					return false;
```
Comment (urkerab) [inline data/moves.ts:6313]: Do you need to track a success flag instead?

**Comment by Zarel** [comment]
Looks good! This can be fixed once the merge conflict is addressed.

**Comment by KrisXV** [comment]
did @Zarel

**Comment by Zarel** [comment]
tyty

**Pull request merged**